deploy/python/infer.py

```diff
@@ -536,33 +343,67 @@ def predict_video():
         im = visualize_box_mask(
             frame,
             results,
-            detector.config.labels,
-            mask_resolution=detector.config.mask_resolution)
+            detector.pred_config.labels,
+            mask_resolution=detector.pred_config.mask_resolution,
+            threshold=FLAGS.threshold)
         im = np.array(im)
         writer.write(im)
+        if camera_id != -1:
+            cv2.imshow('Mask Detection', im)
+            if cv2.waitKey(1) & 0xFF == ord('q'):
+                break
     writer.release()
 
 
+def main():
+    pred_config = PredictConfig(FLAGS.model_dir)
+    detector = Detector(
+        pred_config,
+        FLAGS.model_dir,
+        use_gpu=FLAGS.use_gpu,
+        run_mode=FLAGS.run_mode)
+    # predict from image
+    if FLAGS.image_file != '':
+        predict_image(detector)
+    # predict from video file or camera video stream
+    if FLAGS.video_file != '' or FLAGS.camera_id != -1:
+        predict_video(detector, FLAGS.camera_id)
+
+
 if __name__ == '__main__':
+    paddle.enable_static()
     parser = argparse.ArgumentParser(description=__doc__)
     parser.add_argument(
         "--model_dir",
         type=str,
         default=None,
-        help=("Directory include:'__model__', '__params__', "
+        help=("Directory include:'model.pdiparams', 'model.pdmodel', "
```

Comment:
model.pdiparams  ->  model.pdparams

Reply:
the output formats of jit.save are pdiparams. pdiparams.info and pdmodel